Packs/CyberArkPAS/Integrations/CyberArkPAS/CyberArkPAS.py

```diff
@@ -0,0 +1,1095 @@
+                                            ):
+        url_suffix = f"/passwordvault/api/Accounts/{account_id}/SetNextPassword"
+        body = {
+            "ChangeImmediately": "true" == "true",
+            "NewCredentials": new_credentials,
+        }
+        return self._http_request("POST", url_suffix, json_data=body, resp_type='text')
+
+    def change_credentials_in_vault_only(self,
+                                         account_id: str,
+                                         new_credentials: str,
+                                         ):
+        url_suffix = f"/passwordvault/api/Accounts/{account_id}/Password/Update"
+        body = {
+            "NewCredentials": new_credentials,
+        }
+        return self._http_request("POST", url_suffix, json_data=body, resp_type='text')
+
+    def verify_credentials(self,
+                           account_id: str,
+                           ):
+        url_suffix = f"/PasswordVault/WebServices/PIMServices.svc/Accounts/{account_id}/VerifyCredentials"
+
+        return self._http_request("POST", url_suffix, resp_type='text')
+
+    def reconcile_credentials(self,
+                              account_id: str,
+                              ):
+        url_suffix = f"/PasswordVault/API/Accounts/{account_id}/Reconcile"
+
+        return self._http_request("POST", url_suffix, resp_type='text')
+
+    def update_account(self,
+                       account_id: str,
+                       account_name: str,
+                       address: str,
+                       username: str,
+                       platform_id: str,
+                       ):
+        url_suffix = f"/PasswordVault/api/Accounts/{account_id}"
+
+        arguments = {"name": account_name,
+                     "address": address,
+                     "userName": username,
+                     "platformId": platform_id}
+        body = []
+        for key, value in arguments.items():
+            if value:
+                body.append(
+                    {
+                        "op": "replace",
+                        "path": f"/{key}",
+                        "value": f"{value}"
+                    }
+                )
+
+        return self._http_request("PATCH", url_suffix, data=str(body))
+
+    def get_security_events(self,
+                            next_run: str
+                            ):
+        url_suffix = "/PasswordVault/API/pta/API/Events/"
+        self._headers["lastUpdatedEventDate"] = next_run
+        return self._http_request("GET", url_suffix)
+
+
+def test_module(
+        client: Client,
+) -> str:
+    """
+    If a client was made then an accesses token was successfully reached,
+    therefor the username and password are valid and a connection was made
+    additionally, checks if not using all the optional quota
+    :param client: the client object with an access token
+    :return: ok if got a valid accesses token and not all the quota is used at the moment
+    """
```

Comment:
add here the client function of list_securoty_events that will encapsulate a test for the fetch flow and params